src/sqlancer/postgres/gen/PostgresInsertGenerator.java

```diff
@@ -14,9 +14,9 @@
 import sqlancer.postgres.PostgresVisitor;
 import sqlancer.postgres.ast.PostgresExpression;
 
-public final class PostgresInsertGenerator {
+public class PostgresInsertGenerator {
 
-    private PostgresInsertGenerator() {
+    protected PostgresInsertGenerator() {
```

Comment:
I did this, but I think the maven configurations are unhappy about it:
[ERROR] src/sqlancer/postgres/gen/PostgresDeleteGenerator.java:[14,1] (design) HideUtilityClassConstructor: Utility classes should not have a public or default constructor.
[ERROR] src/sqlancer/postgres/gen/PostgresInsertGenerator.java:[17,1] (design) HideUtilityClassConstructor: Utility classes should not have a public or default constructor.
[ERROR] src/sqlancer/postgres/gen/PostgresUpdateGenerator.java:[18,1] (design) HideUtilityClassConstructor: Utility classes should not have a public or default constructor.
[ERROR] src/sqlancer/postgres/gen/PostgresIndexGenerator.java:[20,1] (design) HideUtilityClassConstructor: Utility classes should not have a public or default constructor.
[ERROR] src/sqlancer/postgres/gen/PostgresSetGenerator.java:[12,1] (design) HideUtilityClassConstructor: Utility classes should not have a public or default constructor.
[ERROR] src/sqlancer/postgres/gen/PostgresCommon.java:[20,1] (design) HideUtilityClassConstructor: Utility classes should not have a public or default constructor.
[ERROR] src/sqlancer/postgres/gen/PostgresViewGenerator.java:[14,1] (design) HideUtilityClassConstructor: Utility classes should not have a public or default constructor.

Reply:
Okay, I see now why. The inheritance relationship does not seem to provide any benefits, since the classes you are inheriting from only have static methods (which is why the are utility classes). Rather than changing these Postgres classes, you can directly call their static methods.